DataManagement/GitTutorial.Rmd

```diff
@@ -260,11 +322,9 @@ In general only data or output files are commited, but exceptions can be useful
 
```

Comment:
previous line - In general only data or output files are commited, - needs a negative somewhere! committed